WooCommerce/WooCommerceTests/Mocks/MockCardPresentPaymentsStoresManager.swift

```diff
@@ -19,6 +19,8 @@ final class MockCardPresentPaymentsStoresManager: DefaultStoresManager {
     private var softwareUpdateSubject: CurrentValueSubject<CardReaderSoftwareUpdateState, Never> = .init(.none)
     private var paymentExtension: CardPresentPaymentGatewayExtension
 
+    var recievedActions: [CardPresentPaymentAction] = []
```

Comment:
Typo here, `recievedActions` -> `receivedActions`

Reply:
Will fix

Reply:
Fixed in 791b53b